api/app/controllers/api/v1/passwords_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Api
+  module V1
+    # Passwords controller
+    class PasswordsController < ApplicationController
+      include Password
+
+      def new
+        user = User.find_by(email: params[:email])
+        return user_not_found_error unless user
+        user.process_reset_password_request
+      end
+
+      def update
+        user = User.by_reset_token(params[:reset_token]).first
```

Comment:
I think I'd roll up the .first into the `by_reset_token` method, unless there's a compelling reason not to.